src/postponed_job.rs

```diff
@@ -0,0 +1,76 @@
+//! Functions for interacting with the postponed jobs API.
+
+use crate::Ruby;
+
+use rb_sys::rb_postponed_job_register;
+use std::ffi::c_void;
```

Comment:
You can match the import style used in this project with the following options in `./rustfmt.toml`

``` toml
imports_granularity = "Crate"
group_imports = "StdExternalCrate"
```

and `cargo +nightly fmt`.

(It's not committed because it generates a bunch of warnings on not-nightly and it's annoying)

Basiaclly it's

``` rust
use std::something;

use dependency_a::foo;
use dependency_b::bar;

use crate::stuff;
```